debian-setup.sh

```diff
@@ -27,6 +27,7 @@ apt-get install -y -qq \
     openssl \
     qemu-system-arm \
     qemu-system-x86 \
+    &>/dev/null
```

Comment:
Hmmm do we want to direct both stderr AND stdout to `/dev/null`? If there is an error during install, we'd probably want to see it.

Reply:
done